src/frontend/scss/templates/courses/cms/_organization_list.scss

```diff
@@ -2,24 +2,15 @@
 $richie-org-list-title-margin: 0 !default;
 $richie-org-list-title-padding: 1rem !default;
 $richie-org-list-title-textalign: center !default;
+$richie-org-list-divider: 1rem !default;
 
 $richie-org-list-content-margin: 0 auto !default;
 $richie-org-list-content-padding: 1rem 0 !default;
 
-$richie-org-list-item-gutter: 0.625rem !default;
-$richie-org-list-item-border-hover: 1px solid $firebrick6 !default;
-
-$richie-org-list-item-title-fontsize: 0.81rem !default;
-$richie-org-list-item-title-fontcolor: $black !default;
-$richie-org-list-item-title-textalign: center !default;
-
-$richie-org-list-item-empty-fontcolor: $gray40;
-$richie-org-list-item-empty-textalign: center;
-
```

Comment:
@sveetch it seems that you forgot the grey background?

Reply:
You meant "white" background, done.